app/scripts/components/common/mapbox/index.tsx

```diff
@@ -131,6 +133,15 @@ function MapboxMapComponent(props: MapboxMapProps, ref) {
   const [isMapLoaded, setMapLoaded] = useState(false);
   const [isMapCompareLoaded, setMapCompareLoaded] = useState(false);
 
+  const {
+    styleLoaded,
+    currentBasemapStyleId,
```

Comment:
Only the basemap has a `current` prefix. Should this be normalized?

Reply:
Yes, will do